website_ranking_procedure.sql

```diff
@@ -0,0 +1,91 @@
+TRUNCATE TABLE  egstats.player_kpm;
```

Comment:
At this point the table will not yet exist so there should be a CREATE TABLE IF NOT EXISTS command somewhere prior that creates the table. This TRUNCATE should probably have a IF EXISTS clause too or if it's meant to run periodically it should be in its own stored procedure.